GPyOpt/core/bo.py

```diff
@@ -149,6 +149,9 @@ def run_optimization(self, max_iter = 0, max_time = np.inf,  eps = 1e-8, context
             self.cum_time = time.time() - self.time_zero
             self.num_acquisitions += 1
 
+            if verbosity:
+                print("num acquisition: %d, time elapsed: %.2fs" % (
```

Comment:
Can you use `"".format()` here? It is a preferred way of doing string formats in python these days

Reply:
Sure thing.